### PR TITLE
build: add sha256sum for release pkg

### PR DESCRIPTION
**Pull request by try-agaaain**
build: add sha256sum for release pkg [#2507](https://github.com/4paradigm/OpenMLDB/issues/2507)

### CODE REVIEW

File: .github/workflows/cicd.yaml [hunk 1]
```diff
@@ -194,6 +195,7 @@ jobs:
         if: ${{ github.event_name == 'push' }}
         run: |
           tar czf ${{ env.OPENMLDB_PREFIX }}.tar.gz ${{ env.OPENMLDB_PREFIX }}/
+          sha256sum ${{ env.OPENMLDB_PREFIX }}.tar.gz > ${{ env.OPENMLDB_PREFIX }}-sha256sum
```
Comment (aceforeverd) [inline .github/workflows/cicd.yaml]: pls ref https://github.com/4paradigm/hybridsql-asserts/blob/main/.github/workflows/build.yml#L162-L184. 

changes only needed in `release` job. And a single `SHA256SUM` file is required for extra release

**Review by aceforeverd — CHANGES_REQUESTED** [review]

File: .github/workflows/cicd.yaml (hunk 1 above)
Comment (try-agaaain) [reply .github/workflows/cicd.yaml]: Does that mean putting the sha256sum of the two installation packages for Linux and drawin in the same file?

Like this:
```yaml
      - name: generate signature
        run: |
          sha256sum openmldb-*.tar.gz > SHA256SUM

      - name: Release
        if: ${{ startsWith(github.ref, 'refs/tags/v') }}
        uses: softprops/action-gh-release@v1
        with:
          files: |
            openmldb-*.tar.gz
            SHA256SUM
```

Comment (aceforeverd) [reply .github/workflows/cicd.yaml]: yes!

**Comment by try-agaaain** [comment]
Here's a check failure, how can I fix it?

**Comment by aceforeverd** [comment]
> Here's a check failure, how can I fix it?

Should be fine to ignore

**Review by aceforeverd — APPROVED** [review]

**Review by vagetablechicken — APPROVED** [review]

**Pull request merged**